accounts/views.py

```diff
@@ -73,6 +76,81 @@ def citesphere_token(request):
         token.save()
         return Response(status=status.HTTP_201_CREATED)
 
+class ForgotPasswordView(APIView):
+    def post(self, request):
+        username = request.data.get('username', None)
+        if username:
+            try:
+                user = VogonUser.objects.get(username=username)
+            except VogonUser.DoesNotExist:
+                return Response({ "success": False, "message": "User not found" }, status=404)
+            token = jwt.encode({ 
+                'exp': arrow.utcnow().shift(days=1).timestamp,
+                'username': user.username,
+            }, settings.SECRET_KEY, algorithm='HS256').decode('UTF-8')
+            reset_token = ResetToken(user=user, token=token)
+            reset_token.save()
+
+            # Email token
+            send_mail(
+                'Vogon password reset',
+                f'Reset link: http://localhost:8080/reset-password/{token}',
```

Comment:
lets make the reset link and the vogon email be configurable from the settings file